### PR TITLE
Update Go to 1.15.6

### PR DESCRIPTION
**Pull request by tklauser**

### CODE REVIEW

**Comment by tklauser** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Comment by tklauser** [comment]
retest-runtime

**Review by rolinh — APPROVED** [review]

File: images/hubble-proto/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright 2020 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-ARG GOLANG_IMAGE=docker.io/library/golang:1.15.5-alpine3.12
+ARG GOLANG_IMAGE=docker.io/library/golang:1.15.6-alpine3.12
```
Comment (kaworu) [inline images/hubble-proto/Dockerfile:4]: Happy to see that hubble-proto hasn't been left out!

**Review by kaworu — APPROVED** [review]

File: images/hubble-proto/Dockerfile (hunk 1 above)
Comment (tklauser) [reply images/hubble-proto/Dockerfile:4]: @kAworu Thanks to your PR #14036 :pray:

**Comment by nebril** [comment]
test-4.9

**Comment by nebril** [comment]
test-4.19

**Pull request merged**